.github/workflows/tags-to-pypi.yml

```diff
@@ -0,0 +1,27 @@
+name: PyPi Publisher
```

Comment:
I haven't seen them set up this way and based the `floris` [version](https://github.com/NREL/floris/blob/master/.github/workflows/python-publish.yml) on the GH Actions [example](https://docs.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries). I noticed that you're using tags instead of releases which is likely why it is different so I'll trust the test version transfers over to the real world correctly and that your know-how are correct here.

Reply:
I merely followed a different guide to set this up: https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

I wonder why the guides are so different between python.org and github.com?

I think it's nice to use tags because they're easy to create (and test). I believe github releases are based on tags.

Reply:
That's fair enough. I assume it would come down to the workflows for whoever created (and approved) the examples.¯\_(ツ)_/¯